codegen/files_test.go

```diff
@@ -12,19 +12,17 @@ func TestHeaderTmpl(t *testing.T) {
 	}{
 		"a minimum header": {
 			data: map[string]interface{}{
-				"Pkg": "endpoints",
+				"Pkg": "foo",
```

Comment:
It's fine in general, the problem is that when you need more than 3 different values you're stuck (`foo`, `bar`, `baz` ??). Just trying to be consistent, not a big deal.